src/errors.rs

```diff
@@ -50,8 +50,11 @@ pub enum OrandaError {
         details: reqwest::Error,
     },
 
-    #[error("Could not find any releases from {repo} with a cargo-dist compatible `dist-manifest.json`.")]
-    NoCargoDistReleasesFound { repo: String },
+    #[error("Could not find any releases from {repo_owner}/{repo_name} with a cargo-dist compatible `dist-manifest.json`.")]
```

Comment:
backticks not terribly useful here..? unless maybe it helps sunfish render..?

Reply:
yeah i want to keep it for sunfish rendering